Makefile

```diff
@@ -376,6 +376,15 @@ release-publish: release-prereqs $(UPLOAD_DIR)
 	@echo "  https://github.com/projectcalico/calico/releases/tag/$(CALICO_VER)"
 	@echo ""
 
+## Updates helm-index with the new release chart
+helm-index: release-prereqs
+	mkdir -p charts/$(CALICO_VER)/
```

Comment:
nit: I'm assuming if something fails (like someone didn't have the helm credentials and this was kicked off) that it's possible that the `charts/` directory will be leftover from a previous run. May want to add a `rm -rf charts` above this line just for safety.

Reply:
```suggestion
	rm -rf  charts
	mkdir -p charts/$(CALICO_VER)/
```